README.md

```diff
@@ -88,14 +88,27 @@ As part of the ARM template, a UI is deployed to `https://{prefix}-ui-{randomStr
 ![Screenshot of sample web app](./images/search-results.png)
 
 ___
-> NOTE: At this point, you should consider [adding authentication to the web app in the Azure portal](https://docs.microsoft.com/en-us/azure/app-service/scenario-secure-app-authentication-app-service) to secure your data.
+> NOTE: At this point, you should consider [adding authentication to the web app in the Azure portal](https://docs.microsoft.com/azure/app-service/scenario-secure-app-authentication-app-service) to secure your data.
 ___
 
 The UI is a React-based Web App available in the `SearchUI` folder. Navigate to [**SearchUI/README.md**](SearchUI/README.md) for more details on the web app. 
 
+## Frequently Asked Questions (FAQ)
+**How is this solution utilizing QnaMaker different from the [new semantic search functionality](https://docs.microsoft.com/azure/search/semantic-search-overview) in Cognitive Search?**
 
-## Disclaimer
+The semantic search question answering does have a functional overlap with this solution accelerator using QnA Maker.  Here's what you need to know.  
++ The QnA extracted by QnA Maker in this solution accelerator are high precision. Also, these QnAs can be edited in the qnamaker.ai portal. 
++ The QnAs provided by the semantic search feature are high recall.  These are extracted at runtime and non-customizable.
```

Comment:
How about something like "The QnAs provided by the semantic search feature are high recall. This gives you the ability to get meaningful answers to much more specific questions. These are extracted at runtime and non-customizable."

Reply:
Added in commit da8dd04b21de120299b9d2aae76ed4a1438912d5.